lib/kernel/cuda/get_local_id.c

```diff
@@ -0,0 +1,38 @@
+/* OpenCL built-in library: get_local_id() for CUDA
```

Comment:
The difference is the type of these external variables. In PTX these special registers are always 32-bits, so we need to use `uint` here instead of `size_t`. I suppose we could use the default `size_t` versions and then hack the LLVM IR later, but I think that would be messier?